src/server/index.js

```diff
@@ -1,4 +1 @@
-require('babel-core/register')({
-    extensions: ['.es6', '.es', '.js']
-});
-require('./server');
+import server from './server';
```

Comment:
Believe this was let over from not using nodemon, but again another good catch! Much cleaner.